content/pages/sphinx.html

```diff
@@ -0,0 +1,243 @@
+<html>
+  <head>
+    <title>Easily build and deploy your Sphinx docs</title>
+    <meta name="template" content="page" />
+    <meta name="slug" content="landing/sphinx" />
+    <meta name="status" content="hidden" />
+    <meta name="description" content="Learn more about Sphinx & Read the Docs hosting" />
+  </head>
+  <body>
+
+{#
+  View this page at http://localhost:8080/landing/sphinx/index.html
+#}
+
+{% import "partials/basic.html" as basic %}
+{% import "partials/marketing.html" as marketing %}
+{% import "partials/technical.html" as technical %}
+
+<section>
+  {% call basic.header_image(
+    header="Quickly deploy your Sphinx docs across multiple versions",
+    image="https://cdn.dribbble.com/users/876899/screenshots/2141696/readthedocs-thumb.png",
+    image_alt="Marketing introduction header",
```

Comment:
I'm not sure if this was a placeholder or it means something that I don't understand 😅